test/native/jest_ui.config.js

```diff
@@ -1,6 +1,3 @@
-/** @flow
- * @format */
-
 const main = require( './jest.config.js' );
```

Comment:
Out of curiosity, why are there 2 Jest config files?

Reply:
If you ask about `jest_ui.config.js` and `jest.config.js`:
- `jest_ui.config.js` - is for e2e tests
- `jest.config.js` - is for unit tests

Reply:
I would never guess myself.